roles/olm_operator/tasks/main.yml

```
@@ -151,8 +151,8 @@
         - sub.resources is defined
         - sub.resources | length == 1
         - "'status' in sub.resources[0]"
-        - "'currentCSV' in sub.resources[0].status"
-        - sub.resources[0].status.currentCSV | length > 0
+        - "'installedCSV' in sub.resources[0].status"
```

Shouldn't check if `installedCSV` is defined?
```suggestion
        - sub.resources[0].status.installedCSV is defined
```